app/views/datagrid/_enum_checkboxes.html.erb

```diff
@@ -2,10 +2,12 @@
 Indent in this file may cause extra space to appear.
 You can add indent if whitespace doesn't matter for you
 %>
+<div class="datagrid-enum-checkboxes">
 <%- elements.each do |value, text, checked| -%>
 <%- id = [form.object_name, filter.name, value].join('_').underscore -%>
-<%= form.label filter.name, options.merge(for: id) do -%>
-<%= form.check_box(filter.name, {multiple: true, id: id, checked: checked, include_hidden: false}, value.to_s, nil) -%>
+<%= form.datagrid_label(filter.name, **options, for: id, class: 'datagrid-enum-checkbox-label') do -%>
```

Comment:
**Breaking change:**

We have such defined like this in view:
```ruby
= f.datagrid_filter :selected_segmentations, class: "js-segmentations"
```

`js-segmentations` class is no longer added to `<label>`.

Reply:
Fixed.